homeassistant/components/vesync/light.py

```diff
@@ -68,17 +79,88 @@ def turn_on(self, **kwargs):
             brightness = round((brightness / 255) * 100)
             # clamp to 1-100
             brightness = max(1, min(brightness, 100))
-            self.dimmer.set_brightness(brightness)
+            self.device.set_brightness(brightness)
+        # set white temperature
+        if ATTR_COLOR_TEMP in kwargs:
```

Comment:
Check if color mode temp is in supported color modes before entering this block

Reply:
done